src/core/analysis-context.ts

```diff
@@ -480,15 +486,22 @@ export class AnalysisContext {
    * Resolves a URL with this Analyzer's `UrlResolver` or returns the given
    * URL if it can not be resolved.
    */
-  resolveUrl(url: PackageRelativeUrl): ResolvedUrl|undefined {
-    return this.resolver.resolve(url);
+  resolveUrl(
```

Comment:
Need to update the comment here.

Also, is this method paying for itself? It's obscuring finding usages of `UrlResolver.resolve()`

Reply:
+1, removed